### PR TITLE
Update add incident report form

### PR DESCRIPTION
**Pull request by kushaljh**

### CODE REVIEW

**Review by aashnasheth — APPROVED** [review]

File: server/routes.js [hunk 1]
```diff
@@ -597,8 +597,9 @@ app.route("/incidentReport")
...
+		console.log("sigy" + incident.manualSignature);
+        con.query('INSERT INTO INCIDENTS (INVOLVED_NAMES, NOTIFIED_NAMES, OBSERVER_NAMES, FOLLOW_UP, INCIDENT_DATE, TIME, VID, LOCATION, DESCRIPTION, INJURY, INJURY_DESCRIPTION, ER_VISIT, ER_HOSPITAL, POLICE_REPORT, PR_NUMBER, AUTHOR_ID, AUTHOR_SIG, REVIEWER_ID , AUTHOR_DATE) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)',
+            [incident.peopleInvolvedNames.join(", "), incident.notifiedNames.join(", "), incident.observersNames.join(", "), incident.followUp, incident.incidentDate, incident.time, incident.village, incident.location, incident.description, incident.injury, incident.injuryDescription, incident.emergencyRoom, incident.hospital, incident.policeReport, incident.reportNumber, incident.signature, incident.manualSignature, incident.reviewerName, incident.currentDate])
```
Comment (kushaljh) [inline server/routes.js:601]: lets rename `incident.signature` to `incident.authorId`

File: server/routes.js [hunk 2]
```diff
@@ -597,8 +597,9 @@ app.route("/incidentReport")
         incident.injury = (incident.injury == 'true');
         incident.emergencyRoom = (incident.emergencyRoom == 'true');
 		incident.policeReport = (incident.policeReport == 'true');
-        con.query('INSERT INTO INCIDENTS (INVOLVED_NAMES, NOTIFIED_NAMES, OBSERVER_NAMES, FOLLOW_UP, INCIDENT_DATE, TIME, VID, LOCATION, DESCRIPTION, INJURY, INJURY_DESCRIPTION, ER_VISIT, ER_HOSPITAL, POLICE_REPORT, PR_NUMBER, AUTHOR_ID , REVIEWER_ID , AUTHOR_DATE) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)',
-            [incident.peopleInvolvedNames.join(", "), incident.notifiedNames.join(", "), incident.observersNames.join(", "), incident.followUp, incident.incidentDate, incident.time, incident.village, incident.location, incident.description, incident.injury, incident.injuryDescription, incident.emergencyRoom, incident.hospital, incident.policeReport, incident.reportNumber, incident.signature, incident.reviewerName, incident.currentDate])
+		console.log("sigy" + incident.manualSignature);
```
Comment (aashnasheth) [inline server/routes.js]: Remove this line

File: src/IncidentReport.js [hunk 3]
```diff
@@ -26,6 +26,7 @@ class IncidentReport extends Component {
             reportNumber: "",
             peopleNotified: [],
             signature: "",
```
Comment (kushaljh) [inline src/IncidentReport.js:28]: The above would also need to be updated similarly

**Review by aashnasheth — APPROVED** [review]

**Comment by wilsmic2** [comment]
looks good to me

**Review by wilsmic2 — APPROVED** [review]

**Pull request merged**